distribution/zip/pom.xml

```diff
@@ -60,7 +60,11 @@
                         </goals>
                         <configuration>
                             <target if="${run}">
-                                <ant antfile="${elasticsearch.integ.antfile}" target="start-foreground"/>
+                                <!-- use conventional port numbers -->
+                                <ant antfile="${elasticsearch.integ.antfile}" target="start-foreground">
+                                    <property name="integ.http.port" value="9200"/>
```

Comment:
This run elasticsearch in the foreground with the current source code, with remote debugger enabled. its intended for devs to work with their IDE and so on.